crates/store/re_video/src/decode/ffmpeg.rs

```diff
@@ -546,7 +560,32 @@ fn read_ffmpeg_output(
             }
 
             FfmpegEvent::ParsedVersion(ffmpeg_version) => {
-                re_log::debug_once!("FFmpeg version is: {}", ffmpeg_version.version);
+                re_log::debug_once!("FFmpeg version is {}", ffmpeg_version.version);
+
+                let mut version_parts = ffmpeg_version.version.split(".");
+                if let (Some(major), Some(minor)) = (
+                    version_parts
+                        .next()
+                        .and_then(|part| part.parse::<u32>().ok()),
+                    version_parts
+                        .next()
+                        .and_then(|part| part.parse::<u32>().ok()),
+                ) {
+                    if major < FFMPEG_MINIMUM_VERSION_MAJOR
+                        || (major == FFMPEG_MINIMUM_VERSION_MAJOR
+                            && minor < FFMPEG_MINIMUM_VERSION_MINOR)
+                    {
+                        re_log::warn_once!(
+                            "FFmpeg version is {}. Only versions >= {FFMPEG_MINIMUM_VERSION_MAJOR}.{FFMPEG_MINIMUM_VERSION_MINOR} are officially supported",
+                            ffmpeg_version.version
+                        );
+                    }
+                } else {
+                    re_log::warn_once!(
+                        "Failed to parse FFmpeg version: {}",
+                        ffmpeg_version.raw_log_message
+                    );
```

Comment:
I think at this point we should return an error, so that we show the "Install ffmpeg" stuff in the selection panel, perhaps with a different message